internal/provider/registry/registry_test.go

```diff
@@ -302,6 +302,36 @@ var _ = Describe("Test AddPlatformToInstallConfig", func() {
 			Expect(cfg.Platform.Baremetal.Hosts[1].Name).Should(Equal("hostname1"))
 			Expect(cfg.Platform.Baremetal.Hosts[2].Name).Should(Equal("hostname2"))
 		})
+		It("fails for host without interface from machine networks", func() {
```

Comment:
one success case were we validate that we took right mac?

Reply:
Sure, just added a 2nd interface to all the hosts and added a test to confirm that we pick MAC of the interface where address is assigned